pom.xml

```diff
@@ -937,24 +1003,6 @@
           </reportSet>
         </reportSets>
       </plugin>
-      <plugin>
```

Comment:
We don't need to run the `report-aggregate` goal since ide-sidecar is a single module project.